os_dep/osdep_service.c

```diff
@@ -1782,20 +1782,26 @@ static int isFileReadable(char *path)
 { 
 	struct file *fp;
 	int ret = 0;
+#ifdef set_fs
 	mm_segment_t oldfs;
+#endif
 	char buf;
  
 	fp=filp_open(path, O_RDONLY, 0); 
 	if(IS_ERR(fp)) {
 		ret = PTR_ERR(fp);
 	}
 	else {
-		oldfs = get_fs(); set_fs(KERNEL_DS);
-		
+#ifdef set_fs
+		oldfs = get_fs();
+        set_fs(KERNEL_DS);
```

Comment:
minor nitpick: should use tabs for indent instead of spaces here and similar instances below